src/app/page.jsx

```diff
@@ -16,23 +15,44 @@ async function getProjects() {
             updatedAt
           }
         }
-      `,
-    });
-    return result.data
-  } catch (error) {
-    console.error("Error fetching data:", error)
-    return null
+      `;
+
+  const data = await fetchGraphQL(query);
+  return data;
+}
+
+async function getPosts() {
+  const query = `
+  query Posts {
+    posts {
+      createdAt
+      content {
+        text
+      }
+      title
+      publishedAt
+      updatedAt
+      createdBy {
+        name
+      }
+      id
+    }
   }
+`;
+
+  const data = await fetchGraphQL(query);
+  return data;
 }
 
 export default async function Home() {
-  const projects = await getProjects();
+  const { heroes } = await getProjects();
+  const { posts } = await getPosts();
```

Comment:
maybe initiate query by name and use only fetchGraphQL function to get const { heroes } and const { posts }

Reply:
having getProjects and getPosts makes the code easier to read, it's possible to reuse these methods for other pages, and if we ever change how we "getProjects" or "getPosts", we only have to change those functions

Reply:
ok, then done